tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -27,10 +27,63 @@ Grep Logs And Count Lines
     ${linecount}=  Get Line Count  ${output}
     Should Be Equal As Integers  ${linecount}  ${total}
 
+Install VIC with version to Test Server
+  [Arguments]  ${version}=7315
+  Log To Console  \nDownloading vic ${version} from bintray...
+  ${rc}  ${output}=  Run And Return Rc And Output  wget https://bintray.com/vmware/vic-repo/download_file?file_path=vic_${version}.tar.gz -O vic.tar.gz
+  ${rc}  ${output}=  Run And Return Rc And Output  tar zxvf vic.tar.gz
+  Set Environment Variable  TEST_TIMEOUT  20m0s
+  Install VIC Appliance To Test Server  vic-machine=./vic/vic-machine-linux  appliance-iso=./vic/appliance.iso  bootstrap-iso=./vic/bootstrap.iso  certs=${false}
+  Set Environment Variable  VIC-ADMIN  %{VCH-IP}:2378
+  Set Environment Variable  INITIAL-VERSION  ${version}
+
+Clean up VIC Appliance And Local Binary
+  Cleanup VIC Appliance On Test Server
+  Run  rm -rf vic.tar.gz vic
```

Comment:
To simplify this whole thing, we can just move this rm -rf to the end of the install and eliminate the need for this wrapper keyword.